tools/packaging/packaging.psm1

```diff
@@ -1,4 +1,23 @@
 $Environment = Get-EnvironmentInformation
+
+# Import a psd1 and return it.
+function Import-Data
+{
+    param(
+        [string]$Path
+    )
+
+    $ErrorActionPreference = 'Stop'
+
+    $psd1 = Get-Content -Raw -Path $Path
+    [ScriptBlock]$scriptBlock = [ScriptBlock]::Create($psd1)
+    [string[]] $allowedCommands = 'ConvertFrom-StringData'
+    $scriptBlock.CheckRestrictedLanguage($allowedCommands,$null,$false)
+    return $scriptBlock.InvokeReturnAsIs()
```

Comment:
You can use `Import-PowerShellDataFile` - it's more restrictive than it needs to be for pure data, but I believe already covers your use case.